Build/create-sbom.ps1

```diff
@@ -0,0 +1,97 @@
+﻿param(
```

Comment:
>    Do we want to fix the tool versions?

do you mean "fix" as in "lock the version" or "fix" as in "repair"?

If it's "lock", I am inclined do say keep it floating until there is an issue? For most cases I would indeed go with "lock", here it feels like it's overkill? Unless we have a problem with different SDK versions etc.